tableauserverclient/models/property_decorators.py

```diff
@@ -8,6 +8,8 @@
     # In case we are in python 3 the string check is different
     basestring = str
 
+literal_range = range
```

Comment:
I didn't realize I was shadowing the builtin before, I actually prefer just renaming the old variable, maybe call it "valid_int_range" and then let range stay the function

Reply:
Renamed the keyword argument and removed the copy of builtin.